source/NVDAObjects/IAccessible/chromium.py

```diff
@@ -90,21 +90,21 @@ class Document(ia2Web.Document):
 
 	def _get_treeInterceptorClass(self) -> typing.Type["TreeInterceptor"]:
 		shouldLoadVBufOnBusyFeatureFlag = bool(
-			config.conf["virtualBuffers"]["loadChromiumVbufOnBusyState"]
+			config.conf["virtualBuffers"]["loadChromiumVBufOnBusyState"]
 		)
 		vBufUnavailableStates = {  # if any of these are in states, don't return ChromeVBuf
 			controlTypes.State.EDITABLE,
 		}
 		if not shouldLoadVBufOnBusyFeatureFlag:
 			log.debug(
 				f"loadChromiumVBufOnBusyState feature flag is {shouldLoadVBufOnBusyFeatureFlag},"
-				" vbuf WILL NOT be loaded when state of the document is busy."
+				" vBuf WILL NOT be loaded when state of the document is busy."

```

Comment:
Changes to spelling of VBuf in the log messages should probably be in #13851 rather than here.